doc/data/messages/b/bad-mcs-method-argument/details.rst

```diff
@@ -1 +1,4 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+*Metaclass method %s should have %s as first argument*
+  Used when a metaclass method has a first argument named differently than the
+  value specified in valid-classmethod-first-arg option (default to "cls"),
+  recommended to easily differentiate them from regular instance methods.
```

Comment:
Check the generated result, we're already using what's in the checker's message's description automatically. If there's nothing to add we can remove the details.rst altogether

Reply:
The generated result from pylint was "Metaclass class method __new__ should have 'mcs' as first argumentPylint(C0204:bad-mcs-classmethod-argument)"